### ISSUE TITLE
a11y issue - Unable to enable arrow keys for TileGroup

### ISSUE THREAD

**Issue by elarue3824** [carbon-design-system/carbon#2564]
a11y issue - Unable to enable arrow keys for TileGroup

-->

## Detailed description

> Describe in detail the issue you're having.
In the TileGroup component, the RadioTiles are not selectable using arrow keys.  I have attempted attaching a keyboard event to the TileGroup, but it does not function according to a11y guidelines.

> Is this a feature request (new component, new icon), a bug, or a general issue?
A general issue.

> What did you expect to happen? What happened instead? What would you like to see changed?
The RadioTiles in a TileGroup should be selectable using arrow keys when the TileGroup is in focus and visible on the screen.

> What browser are you working in?
Firefox - latest version

> What version of the Carbon Design System are you using?
carbon-components 9.52.0
carbon-components-react 6.50.1

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Softlayer/Gen1

## Steps to reproduce the issue

I installed the react-keyboard-event-handler package and used the KeyboardEventHandler component with the TileGroup carbon component.  This did enable arrow keys, but the arrow keys still functioned when the component was not in focus.  Attempts to wrap the KeyboardEventHandler around the TileGroup or the RadioTiles failed to reproduce the desired functionality

**Comment by abbeyhrt** [comment]
@dakahn did we decide if this was a functionality that we need to have for a11y or is it more of a feature request?

**Comment by asudoh** [comment]
@abbeyhrt One question that would answer your question is; Does tab key work to select different radio buttons?

**Comment by abbeyhrt** [comment]
@asudoh yeah, it does but it seems to focus both the label and the input (example [here](https://codesandbox.io/s/codesandbox-8sb8o)) so work could be done around that and we could fold it into this issue, if we wanted? DA, Ale, and I went over this issue together and DA spoke with IBMa to see if we needed to add the arrow keys specifically so I was following up with that conversation as well

**Comment by asudoh** [comment]
I see, thank you for taking a look @abbeyhrt! Sounds an enhancement request wrt keyboard navigation. As I navigate the items, VO doesn't seem to announce the content, though, so a separate issue may need to be filed.

**Comment by abbeyhrt** [comment]
Hi! @elarue3824 Looking into this, it would be an a11y feature request on our side. We spoke to IBMa and they confirmed it isn't an a11y violation since the tiles can be navigated through with tabbing. As this is v9 of carbon, we aren't adding any new features. Have you encountered this problem in v10?

**Comment by abbeyhrt** [comment]
Thanks for looking into this @asudoh, the VO did announce the content for me, at least on the sandbox I provided, was that where it was failing for you?

**Comment by asudoh** [comment]
I could make VO announce the accordion content by using VO-specific keyboard shortcut (instead of tab key). So my apologies @abbeyhrt please ignore my VO thing.

**Comment by abbeyhrt** [comment]
Ah okay, thanks for looking into it more! I'm going to close this issue then as it is a v9 accessibility feature request and we are no longer adding new features to v9. Feel free to open it back up if anyone feels there is work that needs to be addressed!
